### ISSUE TITLE
logger repeats itself after first run

### ISSUE THREAD

**Issue by ali0une** [pengfeng/ask.py#27]
Hi there.

Love how this application is updating, nice to have a gradio Ui :+1: 

Found a bug with the get_logger function : logger works fine on first run, but on second run it repeat itself twice, and on third run it repeats itself trice (it's like twice but once more) ... and so on.

i've fixed it by modifying the get_logger function like this : 
```
def get_logger(log_level: str) -> logging.Logger:
    logger = logging.getLogger(__name__)
    logger.setLevel(log_level)
    handler = logging.StreamHandler()
    formatter = logging.Formatter("%(asctime)s - %(levelname)s - %(message)s")
    handler.setFormatter(formatter)
    if len(logger.handlers) == 0 :
      logger.addHandler(handler)
    return logger
```

Also there is a typo [here](https://github.com/pengfeng/ask.py/blob/26f2e444d1584a86c3c1c4b74a44e698fbe42f56/ask.py#L457) it's _generating_.

**Comment by leettools-dev** [comment]
Great catch! Yes, that was a bug when transitioning from one-and-done to a long running svc. Will add the fix. Thanks!

**Comment by pengfeng** [comment]
Great catch! Yes, that was a bug when transitioning from one-and-done to a long running svc. Will add the fix. Thanks!

**Comment by pengfeng** [comment]
Fixed in #28 .
